Assets/Scripts/FeedingSystem/DogFoodBowl.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections;
```

Comment:
Add your name and brief description of the class above the `using` statements in a multiline comment